go.mod

```diff
@@ -6,11 +6,12 @@ require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/gogo/protobuf v1.3.1
 	github.com/google/go-cmp v0.5.2
-	google.golang.org/grpc v1.27.0
-	k8s.io/api v0.19.3
-	k8s.io/apimachinery v0.19.3
-	k8s.io/client-go v0.19.3
-	k8s.io/code-generator v0.19.3
+	github.com/maistra/xns-informer v0.0.0-20210602150316-33463c1f7d6d
+	google.golang.org/grpc v1.28.1
+	k8s.io/api v0.19.2
```

Comment:
Is this downgrade necessary? Will it impact istio's go.mod/sum?

Reply:
These changes are consistent with what's in xns-informer, which is why I made them.

Reply:
I had forgot we bumped maistra/istio to 1.9. Which means everything should bump to 0.20.x (version that istio uses).

Reply:
I'll update this once maistra/xns-informer#14 is merged

Reply:
I've rebased and updated all of these to v0.20.1, same as xns-informer.